docker-compose.yml

```diff
@@ -29,11 +39,24 @@ services:
       - ./db/schema.sql:/docker-entrypoint-initdb.d/schema.sql
       - db:/var/lib/mysql
 
+  mariatestdb:
+    image: mariadb:10.3
+    command: '--sql_mode="ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"'
+    environment:
+      MARIADB_RANDOM_ROOT_PASSWORD: '1'
+      MARIADB_USER: twfytest
+      MARIADB_PASSWORD: password
+      MARIADB_DATABASE: twfytest
+    volumes:
+      - ./db/schema.sql:/docker-entrypoint-initdb.d/schema.sql
+      - testdb:/var/lib/mysql
+
```

Comment:
I agree that a container-per-database is fine if that's what makes sense for the tests.